docs/src/main/asciidoc/openapi-swaggerui-guide.adoc

```diff
@@ -284,6 +284,87 @@ quarkus.smallrye-openapi.path=/swagger
 
 Hit `CTRL+C` to stop the application.
 
+== Loading OpenAPI Schama From Static Files
+
+Instead of dynamically creating OpenAPI schemas from annotation scanning, Quarkus also supports serving static OpenAPI documents.
+The static file to serve must be a valid document conforming to the https://swagger.io/docs/specification[OpenAPI specification].
+An OpenAPI document that conforms to the  OpenAPI Specification is itself a valid JSON object, that can be represented in `yaml` or `json` formats.
+
+To see this in action, we'll put OpenAPI documentation under `META-INF/openapi.yaml` for our `/fruits` endpoints.
+Quarkus also supports alternative <<open-document-paths, OpenAPI document paths>> if you prefer.
+
+[source, shell]
+----
+openapi: 3.0.1
+info:
+  title: Static OpenAPI document of fruits resource
+  description: Fruit resources Open API documentation
+  version: "1.0"
+
+servers:
+  - url: http://localhost:8080/openapi
+    description: Optional dev mode server description
+
+paths:
+  /fruits:
+    get:
+      responses:
+        200:
+          description: OK - fruits list
+          content:
+            application/json: {}
+    post:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Fruit'
+      responses:
+        200:
+          description: new fruit resource created
+          content:
+            application/json: {}
+    delete:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Fruit'
+      responses:
+        200:
+          description: OK - fruit resource deleted
+          content:
+            application/json: {}
+components:
+  schemas:
+    Fruit:
+      properties:
+        description:
+          type: string
+        name:
+          type: string
+----
+By default, a request to `/openapi` will serve the combined OpenAPI document from the static file and the model generated from application endpoints code.
+We can however change this to only serve static OpenAPI document by adding `mp.openapi.scan.disable=true` configuration into `application.properties`.
+
+Now, a request to `/openapi` endpoint will serve the static OpenAPI document instead of the generated one.
+
+[[open-document-paths]]
+[TIP]
+.About OpenAPI document paths
+====
+Quarkus supports various file paths for the static OpenAPI document as listed below.
```

Comment:
hum I don't see the recommended location, is that a regression? or GitHub weirdness?

Reply:
Switching work stations "without-syncing-everything" weirdness. :-) Should be good now.

Reply:
Done.